src/formatDate.js

```diff
@@ -50,7 +50,67 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dayDescription = 'DD';
+  const monthDescription = 'MM';
+  const yearDescription = 'YY';
+  const fullYearDescription = 'YYYY';
+
+  const indexOfDevider = 3;
```

Comment:
such fixed values are better to write in uppercase format like 'DAY_DESCRIPTION', 'INDEX_OF_DEVIDER'

When you will be working on project such constants will be kept outside the function (for example in separate file)